include/irr/asset/IMeshPacker.h

```diff
@@ -59,27 +62,39 @@ class IMeshPacker
 
 public:
 
-    IMeshPacker(const SVertexInputParams& preDefinedLayout, const AllocationParams& allocParams, uint16_t maxIndexCountPerMDIData = std::numeric_limits<uint16_t>::max())
-        :m_maxIndexCountPerMDIData(maxIndexCountPerMDIData), /*TODO: delete when all alocators are done: */m_alctrReservedSpace{nullptr, nullptr, nullptr, nullptr}
+    IMeshPacker(const SVertexInputParams& preDefinedLayout, const AllocationParams& allocParams, uint16_t minTriangleCountPerMDIData, uint16_t maxTriangleCountPerMDIData)
```

Comment:
a static method to help me figure out what my `preDefinedLayout` need to be and how many of them I need and how to pair my meshes would be nice... but that's last on the TODO list